content/modules/bpm/pages/jmix-view.adoc

```diff
@@ -1,6 +1,7 @@
-= Jmix Screen
+= Jmix View
+:page-aliases: jmix-screen.adoc
 
-When you need a process form with a complex layout and behavior, you can use an existing Jmix UI screen instead of input dialog forms. A screen controller should be annotated with the `@ProcessForm` annotation to be used as a process form.
+When you need a process form with a complex layout and behavior, you can use an existing Jmix UI view instead of input dialog forms. A screen controller should be annotated with the `@ProcessForm` annotation to be used as a process form.
```

Comment:
Replace "screen" with "view" everywhere in your edited sections. Also please update the root "Process Forms" section.